dynamics-nav/Sessions.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Sessions"
+description: Use the session functions to manage navnow sessions.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.